### DISCUSSION TITLE
Mobile version for monitoring and/or notification

### DISCUSSION THREAD

**Discussion by TheJonaz** [madMAx43v3r/chia-plotter#944] [Ideas]
Are there any plans on a mobile app, to monitor your plots, hardware space and maybe get notification on blocks found, drive issues etc? Not to use for plotting, but more monitoring.

**Comment by baerrs** [comment]
I've come up with a really dirty solution. I'm monitoring the output of the madmax's program,(along with output from chia via RPC, and monitoring its log files) and push it to my MQTT broker.. I then have a webpage that displays this information via javascript.
![Screenshot_20210924-015320_Chrome](https://user-images.githubusercontent.com/56986668/134626595-8288fbe1-2706-4119-ae41-49176a901512.jpg)

**Comment by altendky** [comment]
I haven't used these or vetted them but they get mentioned regularly.  If you use plotman to schedule your plots (for chiapos, madMAx, or BladeBit) it has prometheus output that might be able to go somewhere useful.  Or you could add another output format if needed.  I believe there are several other monitoring tools.

- https://github.com/martomi/chiadog
- https://github.com/joaquimguimaraes/farmr

**Comment by hajes** [comment]
yes Munin used on any Linux server ;-)

or writing simple scripts.

rest is in Chia GUI or CLI output
